### PR TITLE
Newsletters: add 185 (2022-02-02)

### PR DESCRIPTION
**Pull request by harding**
- [x] Topics, RCs, and releases update @harding 
- [x] Bitcoin Core 23201 wallet: Allow users to specify input weights when funding a transaction @glozow 
- [x] Eclair 2141 More aggressive confirmation target when low on utxos @dongcarl 
- [x] BTCPay Server 3341 Add configurable BOLT11 Expiration for refund @bitschmidty

### CODE REVIEW

File: _posts/en/newsletters/2022-02-02-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23201][] improves the ability of wallet users to fund
+  transactions with external inputs (previously mentioned in [Newsletter
+  #170][news170 external inputs]) by allowing them to specify maximum
+  weights instead of solving data.  This enables applications to use
+  `fundrawtransaction`, `send`, and `walletfundpsbt` RPCs to fee bump
+  transactions with nonstandard outputs such as HTLCs, a requirement for
+  Lightning clients described in [Newsletter #184][news184 eclair auto bump]).
```
Comment (glozow) [inline _posts/en/newsletters/2022-02-02-newsletter.md:114]: @harding I felt that linking to the summary of #17211 would be more relevant here, but feel free to remove.

File: _posts/en/newsletters/2022-02-02-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  operations.
+
+  In brief, for each possible terminal state of a contract---e.g., Alice
+  gets 1 BTC, Bob gets 2 BTC---DLCs currently require creating a
+  separate [signature adaptor][topic adaptor signatures] for that state.
+  Many contracts define a large number of possible terminal states, such
+  as a contract about the future price of bitcoins which specify prices
+  rounded to the nearest dollar and need to cover several thousand
+  dollars worth of price range even for a relatively short-term
+  contract.  That requires the participants create, exchange, and store
```
Comment (glozow) [inline _posts/en/newsletters/2022-02-02-newsletter.md:34]:
```suggestion
  contract.  That requires the participants to create, exchange, and store
```

File: _posts/en/newsletters/2022-02-02-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  later opcodes to use.  A new [OP_CHECKSIGFROMSTACK][topic
+  op_checksigfromstack] (CSFS) opcode (as previously proposed) would
+  allow a tapscript to specify a public key and require a corresponding
+  signature over particular data on the stack---such as the computed
+  digest of the transaction created by `OP_TXHASH`.
+
+    O'Connor explained how these two opcodes would allow emulation of
+    two earlier soft fork proposals, [SIGHASH_ANYPREVOUT][topic
+    sighash_anyprevout] (APO, specified in [BIP118][]) and
+    [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] (CTV
```
Comment (glozow) [inline _posts/en/newsletters/2022-02-02-newsletter.md:68]:
```suggestion
    [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] (CTV,
```

File: _posts/en/newsletters/2022-02-02-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    specified in [BIP119][]).  For some purposes, the emulation might be
+    less efficient than using CTV or APO directly, but `OP_TXHASH` and
+    `OP_CSFS` would keep the Tapscript language simpler and provide more
+    flexibility for future script writers, especially if it was combined
```
Comment (glozow) [inline _posts/en/newsletters/2022-02-02-newsletter.md:72]:
```suggestion
    flexibility for future script writers, especially if combined
```

**Review by glozow — COMMENTED** [review]
Looks great, just have some really minor comments.

**Comment by adamjonas** [comment]
Couldn't catch anything that 🦅 eye Glozow didn't see.

**Review by Xekyo — COMMENTED** [review]
LGTM, no nits

**Comment by harding** [comment]
Pushed edits (thanks @glozow @adamjonas @Xekyo !) and reviewed contributions from @dongcarl @glozow @bitschmidty (thanks!).

**Comment by bitschmidty** [comment]
Thanks to the team for another great newsletter! 🙏 @harding @glozow @Xekyo @dongcarl @adamjonas

**Pull request merged**